package.json

```diff
@@ -15,31 +15,31 @@
     "check-types": "tsc --noEmit"
   },
   "dependencies": {
-    "@next/font": "^13.2.1",
-    "@phntms/css-components": "^0.2.0",
+    "@next/font": "^13.3.0",
```

Comment:
I'm pretty sure @next/font is deprecated, think it can just be imported from "next/font" now